docs/migration/manual-migration.mdx

```diff
@@ -11,36 +11,71 @@ This manual migration process should be run after the
 [automated migration process](./automated-migration), to complete the missing
 parts, or debug issues in the migration CLI output.
 
+## Components
+
+### `HomeSeo`
+
+The `HomeSeo` component has been removed in favor of
+[Remix Meta](https://remix.run/docs/en/main/route/meta-v2)
+
+We have introduced a new Meta generation helper `generateMetas`.
+
+This helper will do more things :
+
+- Merge with parents Metadata
+- Will allow easier canonical generation
+
+The first arguments takes a
+[Remix V2 Meta Function](https://remix.run/docs/en/main/route/meta-v2#meta-function-parameters)
+
+The second arguments takes a canonical path (base url will be automatically
+injected)
+
+```jsx
+export const meta: V2_MetaFunction = (args: any) => {
+  const metas = generateMetas(() => [{ title: "Welcome home" }], "/");
+  return metas(args);
+};
+```
+
 ## `makeCommandDispatcherWithCommandFeedback` API change
 
-[`makeCommandDispatcherWithCommandFeedback` API has
-changed](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/commit/2cc5515682349ca0f1b1aba922a277db27ad067c?merge_request_iid=2200).
-In version 2, the second parameter of `makeCommandDispatcherWithCommandFeedback` was an
-optional boolean (default `true`) and if this parameter was evaluated to a
-truthy value, the resulting component had the responsibility of displaying the
+[`makeCommandDispatcherWithCommandFeedback` API has changed](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/commit/2cc5515682349ca0f1b1aba922a277db27ad067c?merge_request_iid=2200).
+In version 2, the second parameter of `makeCommandDispatcherWithCommandFeedback`
+was an optional boolean (default `true`) and if this parameter was evaluated to
+a truthy value, the resulting component had the responsibility of displaying the
```

Comment:
**thought:** Those changes aren't normal. It seems like those lines were prettified _the wrong way_. It might be a good call to notice the author that they might have an issue with they prettier configuration.

Reply:
It seems a config problem by my side, fixed it